app/controllers/api/v1/packages_controller.rb

```diff
@@ -14,7 +16,7 @@ class PackagesController < Api::V1::ApiController
 
     def_param_group :package do
       param :package, Hash, required: true do
-        param :quantity, Integer, desc: "Package quantity", allow_nil: true
+        param :quantity, lambda { |val| [String, Fixnum].include? val.class }, desc: "Package quantity", allow_nil: true
```

Comment:
@swatijadhav how are we getting a Fixnum here? params should always be strings. Was it a test?